resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -58,7 +58,17 @@
                         @if(isset($tickets))
                             @foreach($tickets as $t)
                                 <tr>
-                                    <td><center><a href="/dashboard/controllers/ticket/{{ $t->id }}">{{ $t->date }}</a></center></td>
+                                    <td>
+                                        <center>
+                                            <a href="/dashboard/controllers/ticket/{{ $t->id }}">{{ $t->date }}</a>
+                                            <br />
+                                            @if($t->student_comments)
```

Comment:
These badges are backwards. @if($t->student_comments) should result in "Comments Submitted"

Reply:
Haha that would help. Resolved and squashed into 3547c35